src/components/Compounder/CompounderVaultTable.tsx

```diff
@@ -43,7 +43,7 @@ export const CompounderVaultTable: FC<CompounderVaultTableProps> = ({
   const showLoadingState = compoundersList.isLoading || !clientReady
 
   return (
-    <VaultTable label={`${displayName} Compounders`}>
+    <VaultTable label={`${displayName} Compounders`} earningEnabled>
```

Comment:
EarningEnabled is to enable/disable the earning column - the concentrator doesn't have the earning column